sdv/metadata/dataset.py

```diff
@@ -419,64 +419,14 @@ def _get_pii_fields(self, table_name):
 
         return pii_fields
 
-    @staticmethod
-    def _get_transformers(dtypes, pii_fields):
-        """Create the transformer instances needed to process the given dtypes.
-
-        Temporary drop-in replacement of ``HyperTransformer._analyze`` method,
-        before RDT catches up.
-
-        Args:
-            dtypes (dict):
-                mapping of field names and dtypes.
-            pii_fields (dict):
-                mapping of pii field names and categories.
-
-        Returns:
-            dict:
-                mapping of field names and transformer instances.
-        """
-        transformers_dict = dict()
-        for name, dtype in dtypes.items():
-            dtype = np.dtype(dtype)
-            if dtype.kind == 'i':
-                transformer = transformers.NumericalTransformer(dtype=int)
-            elif dtype.kind == 'f':
-                transformer = transformers.NumericalTransformer(dtype=float)
-            elif dtype.kind == 'O':
-                anonymize = pii_fields.get(name)
-                transformer = transformers.CategoricalTransformer(anonymize=anonymize)
```

Comment:
I can't even find this `anonymize` parameter in the currently used versions of rdt